include/common/configure.h

```diff
@@ -25,6 +25,7 @@
 #include <mutex>
 #include <optional>
 #include <shared_mutex>
+#include <sys/wait.h>
```

Comment:
Why do we need this header?

Reply:
I think the LSP added it for some reason while I was making some changes. We do not require it and I have removed it.

Reply:
I think the LSP added it for some reason while I was making some changes. We do not require it and I have removed it.